src/tests/sphmultipcqueue_t.c

```diff
@@ -888,6 +906,12 @@ main(int argc, char *argv[]) {
 	int test_id,i;
 	int num_producers, num_consumers;
 
+	for (i = 1; i < argc; i++) {
+		cpu_list[i-1] = strtol(argv[i],0,0);
+		if (cpu_list[i-1] > cpu_max) cpu_max = cpu_list[i-1];
```

Comment:
Same question. should we have a check to limit  or validate the input numbers passed?